docs/windows.md

```diff
@@ -201,6 +254,15 @@ Download and apply `antrea-windows.yml`.
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows.yml
 ```
 
+Since Antrea 1.10, you can also deploy antrea-agent Windows DaemonSet with containerd runtime by applying file `antrea-windows-containerd.yml`.
+
+Download and apply `antrea-windows-containerd.yml`.
+
+```bash
+# Example:
```

Comment:
I feel this `# Example:` can be removed.

Reply:
Removed.